tests/python/pytorch/graphbolt/impl/test_fused_csc_sampling_graph.py

```diff
@@ -706,7 +706,7 @@ def test_in_subgraph_node_pairs_homogeneous():
 
     # Extract in subgraph.
     nodes = torch.LongTensor([4, 1, 3])
-    in_subgraph = graph.in_subgraph(nodes)
+    in_subgraph = graph.in_subgraph(nodes, output_cscformat=False)
```

Comment:
Instead of changing them to false, consider update the tests?

Reply:
There already have dedicated tests for csc format in this test file. These tests for coo will be removed when coo part is cleaned up.